apisix/plugins/skywalking.lua

```diff
@@ -126,6 +126,11 @@ function _M.init()
 
     -- TODO: maybe need to fetch them from plugin-metadata
     local metadata_shdict = ngx.shared.tracing_buffer
+
+    if local_plugin_info.service_instance_name == "__hostname__" then
```

Comment:
The variable style that we use in APISIX always with the prefix `$`. What about `$hostname`.